.gitignore

```diff
@@ -33,3 +33,9 @@
 
 # vscode
 .vscode/
+
+# validator analysis
+*.log
+
+# temp files created by the pruner
+*_stripped.p4
```

Comment:
We can use a broader pattern here instead of just temp.p4

Reply:
Shouldn't we also add temp.p4 to gitignore